hie-plugin-api/Haskell/Ide/Engine/Cradle.hs

```diff
@@ -14,8 +33,587 @@ import HIE.Bios as BIOS
 -- of the project that may or may not be accurate.
 findLocalCradle :: FilePath -> IO Cradle
 findLocalCradle fp = do
-  -- Get the cabal directory from the cradle
   cradleConf <- BIOS.findCradle fp
   case cradleConf of
-    Just yaml -> BIOS.loadCradle yaml
-    Nothing -> BIOS.loadImplicitCradle fp
\ No newline at end of file
+    Just yaml -> fixCradle <$> BIOS.loadCradle yaml
+    Nothing   -> cabalHelperCradle fp
```

Comment:
Does this then replace logic in hie-bios for implicitly finding a cradle with cabal-helper instead? Presumably cabal-helper handles this better then?

Reply:
Yes it does and in theory, yes it does.